recipes/aab/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "aab" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aab-{{ version }}.tar.gz
+  sha256: 1be9cd6dceac28c570df268a2aa20c1935a59043c88c9f4ce88a6d208e007d09
+
+build:
+  skip: true  # [py>=400]
+  entry_points:
+    - aab=aab.cli:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - six
+    - jsonschema
+    - whichcraft
+    - pathlib  # [py<34]
+
+test:
+  imports:
+    - aab
+  commands:
+    - pip check
+    - aab --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/glutanimate/anki-addon-builder
+  summary: Anki Add-on Builder
+  dev_url: https://github.com/glutanimate/anki-addon-builder
+  license: AGPL-3.0
```

Comment:
Please check for only vs or-later

Reply:
I don't understand, what do you mean with "check for only vs or-later"?

Reply:
These are the two valid forms of the AGPL-3.0 license identifier, either `AGPL-3.0-only` or `AGPL-3.0-or-later`.

Reply:
done as AGPL-3.0-or-later